packages/asset-service/src/service/AssetService.ts

```diff
@@ -64,4 +64,13 @@ export class AssetService {
       ? this.flatAssetData.filter((asset) => asset.network == network)
       : this.flatAssetData
   }
+
+  async description(name: string, tokenId?: string): Promise<string> {
+    const isToken = !!tokenId
+    const contractUrl = isToken ? `contract/${tokenId?.toLowerCase()}` : ''
+    const { data } = await axios.get(
+      `https://api.coingecko.com/api/v3/coins/${name.toLowerCase()}/${contractUrl}`
```

Comment:
does this request work with a trailing `/` and no contract URL?

Reply:
ya it works